.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*/   @eznix86
```

Comment:
Github is complaining the write access, please look at it
<img width="385" alt="Screenshot 2023-11-14 at 20 15 52" src="https://github.com/zurvan-lab/IPNN/assets/26553194/58cf3c72-da3a-49f8-8234-63902362fdb1">

Reply:
I think we should always put the newline as a last character (github is complaining with ⛔ at the end of the file)